imports/plugins/core/router/lib/router.js

```diff
@@ -87,6 +88,32 @@ class Router {
     return current.query && current.query[name] || undefined;
   }
 
+  static setQueryParams(newParams) {
+    const current = Router.current();
+    const queryParams = Object.assign({}, current.query, newParams);
+
+    for (const key in queryParams) {
+      if (queryParams[key] === null || queryParams[key] === undefined) {
+        delete queryParams[k];
+      }
+    }
+
+    Router.go(current.route.name, current.params, queryParams);
+  }
+
+  static setQueryParams(newParams) {
```

Comment:
Did you intend to have two implementations of `setQueryParams` here? Was maybe one supposed to be `setQueryParam`?